From a54d23542406d35d266a708f362024691209abd5 Mon Sep 17 00:00:00 2001
From: Thomas Goirand <zigo@debian.org>
Date: Wed, 25 Apr 2018 10:41:23 +0200
Subject: [PATCH 01/18] Fix build with Sphinx 1.7.x

In sphinx 1.7, the sphinx.util.compat.Directive class doesn't exist
anymore, it's available in docutils.parsers.rst.Directive instead,
and it was deprecated in Sphinx 1.6 already. This patch makes it
therefore work in both version 1.6 and 1.7 just by using the new
module path docutils.parsers.rst.Directive.

Change-Id: I2e05c1612c687c029174f045d3ea0fec83e1119a
---
 doc/ext/ext.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/ext/ext.py b/doc/ext/ext.py
index 9b07ab1..e0bf27b 100644
--- a/doc/ext/ext.py
+++ b/doc/ext/ext.py
@@ -19,7 +19,7 @@ import os
 from docutils import nodes
 from docutils.statemachine import StringList
 from docutils.parsers.rst.directives import flag, unchanged
-from sphinx.util.compat import Directive
+from docutils.parsers.rst import Directive
 from sphinx.util.nodes import nested_parse_with_titles
 
 from .parser import parse_parser, parser_navigate
-- 
2.10.2

